gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js

```diff
@@ -1255,10 +1255,16 @@ GbGradeTable.modelIndexForStudent = function(studentId) {
 };
 
 
-GbGradeTable.colForAssignment = function(assignmentId) {
-  return GbGradeTable.findIndex(GbGradeTable.instance.view.settings.columns, function(column) {
-           return column._data_ && column._data_.assignmentId === parseInt(assignmentId);
-         });
+GbGradeTable.colForAssignment = function(assignmentId, array) {
+    if (array === undefined){
+        return GbGradeTable.findIndex(GbGradeTable.instance.view.settings.columns, function(column) {
+            return column._data_ && column._data_.assignmentId === parseInt(assignmentId);
+        });
+    } else {
+        return GbGradeTable.findIndex(array, function(column) {
+            return column.assignmentId && column.assignmentId === parseInt(assignmentId);
```

Comment:
I'd add the radix in here, just to be sure, on both parseInts. I know it wasn't there before, but I think it should be there.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Description